### PR TITLE
Remove HTTPS Everywhere component test

### PR DESCRIPTION
**Pull request by stephendonner**
Fixes https://github.com/brave/qa-resources/issues/554

### CODE REVIEW

**Comment by LaurenWags** [comment]
The reference to this component should be removed from all templates. It is listed in https://github.com/brave/qa-resources/blob/master/WikiTemplate/Desktop/Major_CR_Bump/wikitemplate-majorChromiumBumpUpgrades.md as well, possibly others.

**Comment by stephendonner** [comment]
> The reference to this component should be removed from all templates. It is listed in https://github.com/brave/qa-resources/blob/master/WikiTemplate/Desktop/Major_CR_Bump/wikitemplate-majorChromiumBumpUpgrades.md as well, possibly others.

Thanks; searched GitHub and local files, and I've added the above removal, now, which was the only other one left.

**Review by LaurenWags — APPROVED** [review]
++

**Review by MadhaviSeelam — APPROVED** [review]

**Review by MadhaviSeelam — APPROVED** [review]
++

File: WikiTemplate/Desktop/wikitemplate.md [hunk 1]
```diff
@@ -193,7 +193,7 @@
   - [ ] `CertificateRevocation`
   - [ ] `cffkpbalmllkdoenhmdmpbkajipdjfam`: `rs-ABPFilterParserData.dat` & `regional_catalog.json` (AdBlock)
   - [ ] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
-  - [ ] `oofiananboodjbbmdelgdommihjbkfag`: HTTPSE
+  - [ ] `jicbkmdloagakknpihibphagfckhjdih`: `speedreader-updater.dat`
```
Comment (stephendonner) [inline WikiTemplate/Desktop/wikitemplate.md:196]: I have no idea where this change snuck in from, but are they ok to sneak in, so I don't have to rebase/fix merge conflicts yet again, @LaurenWags ?

Comment (LaurenWags) [reply WikiTemplate/Desktop/wikitemplate.md:196]: Is it accurate? Not sure where this one came from either. FWIW I'm not seeing this one on a fresh install of 1.68.x - so we should be careful about adding something that is wrong.

**Pull request closed without merging**